src/lib/bootloader/grub2_widgets.rb

```diff
@@ -1070,7 +1012,7 @@ def contents
         VBox(
           VSpacing(1),
           MarginBox(1, 0.5, KernelAppendWidget.new),
```

Comment:
Here it should also use Grub2KernelAppendWidget, not?

Reply:
That's a BIG bug. Thanks !